datafusion/physical-plan/src/empty.rs

```diff
@@ -229,4 +243,62 @@ mod tests {
         assert!(empty.execute(20, task_ctx).is_err());
         Ok(())
     }
+
+    #[test]
```

Comment:
You can find the partition statistics in the dedicated file: https://github.com/apache/datafusion/blob/main/datafusion/core/tests/physical_optimizer/partition_statistics.rs. 

It also contains the real exection to check the results